### PR TITLE
Changes the receipt email subject

### LINKED ISSUE #945: Change subject of receipt email to "MITx Online Purchase"
Change subject of receipt email to "MITx Online Purchase"

image

### PR DESCRIPTION
**Pull request by jkachel**
#### Pre-Flight checklist

- [X] Testing
  - [X] Code is tested
  - [X] Changes have been manually tested

#### What are the relevant tickets?

Fixes #945

#### What's this PR do?

Changes the email subject on the receipt emails to "MITx Online Receipt". 

#### How should this be manually tested?

Restart your environment (so the change gets picked up) and check out. The receipt email should be "MITx Online Receipt".

### CODE REVIEW

**Comment by pdpinch** [comment]
Having 2nd thoughts about this text. Please don't merge until I've reviewed.

**Review by asadali145 — APPROVED** [review]
LGTM!

**Comment by asadali145** [comment]
@jkachel you can merge this one. Just a reminder in case you forgot to merge as it happened to me in past as well :)

**Comment by jkachel** [comment]
@pdpinch are there changes that need to be made to the wording?

File: ecommerce/templates/mail/product_order_receipt/subject.txt [hunk 1]
```diff
@@ -1 +1 @@
-Purchase Order Receipt
\ No newline at end of file
+MITx Online Purchase
```
Comment (pdpinch) [inline ecommerce/templates/mail/product_order_receipt/subject.txt:1]: I looked through all the email receipts in my inbox, and this pattern seems most common:

```suggestion
Your receipt from MITx Online
```

If `MITx Online` is an available in an environment-specific setting string, we should use that. But not if it's more than 5m of effort.

**Review by pdpinch — CHANGES_REQUESTED** [review]

**Review by asadali145 — APPROVED** [review]

**Review by pdpinch — APPROVED** [review]

**Pull request merged**